docs/src/pages/careers/people-operation-manager.md

```diff
@@ -4,32 +4,33 @@
 
 ## Details of the Role
 
-- Location: Remote (preference for UTC-6 to UTC+5).
-- Type of work: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501))
+- Location: Remote (strong requirement for UTC-6 to UTC+4).
```

Comment:
👍 for updating the time zone as a strong requirement

Reply:
I have excluded India, it wouldn't work.